app/models/import.rb

```diff
@@ -74,6 +83,14 @@ def expected_fields
 
   private
 
+    def header_row?(row)
+      row.none? { |col| numeric?(col) }
```

Comment:
The idea is to assume no header row is present in the CSV file if the first row contains any numeric value. Better suggestions are welcome

Reply:
See my comment above, will resolve this one.